timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -110,21 +110,11 @@ types:
       LockWatchRequest:
         fields:
           references: set<LockWatchReference>
-      NamespacedLeaderTime:
+      Namespace:
+        alias: string
+      LeaderTimes:
         fields:
-          namespace: string
-          leaderTime: LeaderTime
-      NamespacedGetCommitTimestampsRequest:
-        fields:
-          namespace: string
-          numTimestamps: integer
-          lastKnownVersion: optional<ConjureIdentifiedVersion>
-      NamespacedGetCommitTimestampsResponse:
-        fields:
-          namespace: string
-          inclusiveLower: Long
-          inclusiveUpper: Long
-          lockWatchUpdate: LockWatchStateUpdate
+          namespaceWiseLeaderTimes: map<Namespace, LeaderTime>
```

Comment:
can we just have `leaderTimes`? `Map<Namespace, T>` implies it's namespace-wise already